README.md

```diff
@@ -7,7 +7,7 @@
 
 Voilà les instructions pour installer les dépendances sur Arch :
 
-	pacman -S lua51 luarocks5.1
+	pacman -S lua51 luarocks5.1 sassc
```

Comment:
Other parts of the Makefile are referring to sassc and possible packages to install it from. Please edit those lines as well if you’re installing the compiler directly in the dependencies setup instructions.